### PR TITLE
feat: portfolio page re-order & bug fix for resume selector

### LINKED ISSUE #913: File upload opens two select file dialogs when clicked
Windows 10, Microsoft Edge

New file upload #868 opens two select file dialog windows, only using file selected from second one

### PR DESCRIPTION
**Pull request by logan-r**
### Tickets:

- HCK- #913

### List of changes:

- Fix #913, two dialogs open when you clip "Upload resume" box (fixed in #915)
- Add hover effect for resume box
- Change font and spacing for resume box
- Re-order portfolio page so it doesn't feel unbalanced

### Type of change:

Please delete options that aren't relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### PR Checklist:

- [x] Merged `develop` branch (before testing)
- [ ] Linted my code locally
- [x] Listed change(s) in the Changelog
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### Screenshots:
image

### CODE REVIEW

File: src/features/Application/ResumeComponent.tsx [hunk 1]
```diff
@@ -15,13 +15,15 @@ export interface IResumeProps {
...
       <Box ml={props.mode === ManageApplicationModes.EDIT ? '10px' : ''}>
-        <Label>
-          <LabelText label={props.label} required={props.required} />
-          <FileUpload {...props} />
-        </Label>
+        <Box mb={'8px'}>
+          <Label>
+            <LabelText label={props.label} required={props.required} />
+          </Label>
+        </Box>
```
Comment (jamesxu123) [inline src/features/Application/ResumeComponent.tsx:25]: Minor conflict with my earlier PR; I decided to just get rid of the Flex and replace it with a div.

File: src/shared/Form/FileUpload.tsx [hunk 2]
```diff
@@ -30,39 +30,69 @@ export const FileUpload: React.StatelessComponent<
...
+
+            .container:focus {
+              outline: none;
+            }
+            
             .drop-zone {
               width: min(100vw, 960px);
               height: 280px;
-              border: 4px dashed #bcbcbc;
+              border: 4px dashed ${theme.colors.black20};
```
Comment (jamesxu123) [inline src/shared/Form/FileUpload.tsx:57]: Did you mean to change the underlying color? black30 is #bcbcbc

Comment (logan-r) [reply src/shared/Form/FileUpload.tsx:59]: Yeah, went a shade lighter on purpose

File: src/features/Application/ResumeComponent.tsx [hunk 3]
```diff
@@ -15,15 +15,17 @@ export interface IResumeProps {
 const ResumeComponent: React.FC<IResumeProps & FieldProps> = (props) => {
   const viewResume = <ViewPDFComponent {...props} />;
   return (
-    <div style={{ marginBottom: '20px' }}>
+    <Flex mb={'32px'}>
```
Comment (jamesxu123) [inline src/features/Application/ResumeComponent.tsx:18]: I think Flex needs to be imported, it's causing a build error

**Review by jamesxu123 — CHANGES_REQUESTED** [review]

File: src/features/Application/ResumeComponent.tsx (hunk 3 above)
Comment (logan-r) [reply src/features/Application/ResumeComponent.tsx:18]: Yikes, messed up the resolve merge conflicts - was meaning to get rid of Flex

**Review by jamesxu123 — APPROVED** [review]

**Pull request merged**